### PR TITLE
Support Android Firefox (Fenix)

### LINKED ISSUE #8: Install on Firefox mobile
Is there any way to install the Addon in the mobile version of Firefox?
If not, it would be a very much needed feature 👍

### PR DESCRIPTION
**Pull request by heguro**
You can use this method to install nos2x-fox on Android Firefox Nightly:
https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/

For testing on Android:
https://extensionworkshop.com/documentation/develop/developing-extensions-for-firefox-for-android/

There is no `browser.windows` in Android Firefox, so we have to fallback to `browser.tabs`.

Closes #8

EDIT: Added commit for usability on Android

<details><summary>Screenshots</summary>

</details>

### CODE REVIEW

File: src/popup.tsx [hunk 1]
```diff
@@ -56,6 +56,8 @@ function Popup() {
     browser.tabs.create({
       url: browser.runtime.getURL('options.html'),
       active: true
+    }).then(() => {
+      window.close();
```
Comment (diegogurpegui) [inline src/popup.tsx:60]: Why is this here?

Comment (heguro) [reply src/popup.tsx:60]: Without this, we have to manually go back after pressing the Options button to see `options.html`.

<details><summary>Screenshot</summary>



</details>

Comment (diegogurpegui) [reply src/popup.tsx:60]: Got it :+1:

File: src/style.scss [hunk 2]
```diff
@@ -42,6 +42,12 @@ body {
   color: $color-text;
 }
 
+body {
+  margin: 0;
+  padding: 8px;
+  max-width: 100vw;
```
Comment (diegogurpegui) [inline src/style.scss:48]: This style is causing problems in the desktop mode. It causes the popup to open in a weird and annoying way.
Please remove it if it doesn't impact much the mobile version. Or change it in a way that does not affect the desktop version.

Thanks!

Comment (heguro) [reply src/style.scss]: Removed. Thank you for checking.

Comment (diegogurpegui) [reply src/style.scss]: Thanks for fixing. The issue was specifically with the `max-width: 100vw;` line. Feel free to keep tinkering and upload a new PR if you like. I'll merge this one and see if we can make our way into mobile :)

**Review by diegogurpegui — APPROVED** [review]
